xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.h

```diff
@@ -32,6 +32,13 @@ class CDVDSubtitlesLibass : public IDVDResourceCounted<CDVDSubtitlesLibass>
   bool CreateTrack(char* buf, size_t size);
 
 private:
+  /*! \brief Appends a given font from Kodi's font folder (e.g. special://home/media/Fonts/) to be usable by libass.
+   FFMpegDemuxer extracts subtitles to special://temp/fonts and libass is also configured with this font folder. If the user wants to
+   override the ASS font with another kodi configured font, it has to be copied to the temp folder too.
+     \param font The font file to append (e.g. arial.ttf)
+   */
+  void AppendFont(std::string font) const;
```

Comment:
not sure why const is specified here? Also you should be using `const std::string& font`

Reply:
Added const because this method doesn't change any member data. Isn't it a good practice?
I'll add the `const` to the argument too.

Reply:
AFAIK this will only matter if the class object is initialised as const. So no harm here but doesn't actually do anything. Maybe someone else can jump in on why this may be desired or not.

Reply:
Actually this function could be in an anonymous namespace as it doesn't use any class members.